### PR TITLE
SagMeinenNamen.TypingTest version 1.0.0.0

### PR DESCRIPTION
**Pull request by veudal**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/63871)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Looks like a portable app so it requires `InstallerType: portable` but this isn't implemented in the validation pipelines due to outdated WinGet version:
image

If I close the application:
image

You can set this PR to DRAFT and change `InstallerType` from `exe` to `portable`

**Review by veudal — COMMENTED** [review]
LGTM

**Review by ItzLevvie — APPROVED** [review]
It works now ;)

image

Now we're just waiting for the validation pipelines to update to the latest WinGet version so portable apps can be included in the package repository.

**Comment by SagMeinenNamen** [comment]
> waiting for the validation pipelines to update to the latest WinGet version

Do you know how long will this approximately take @ItzLevvie ?

**Comment by ItzLevvie** [comment]
> Do you know how long will this approximately take @ItzLevvie ?

I think @denelon is waiting for most people to update to winget 1.3

**Comment by denelon** [comment]
I would guess we are 5-6 weeks out. We're testing the 1.3 release candidate internally, and we will push it out to Windows Insider Release Preview likely this week.

File: manifests/s/SagMeinenNamen/TypingTest/1.0.0.0/SagMeinenNamen.TypingTest.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+PackageIdentifier: SagMeinenNamen.TypingTest
+PackageVersion: 1.0.0.0
+Installers:
+- Architecture: x86
+  InstallerType: portable
+  InstallerUrl: https://github.com/SagMeinenNamen/TypingTest/raw/main/TypingTest.exe
+  InstallerSha256: E477B4DA4729F733E24C5D9794D48AA36A5B967ECABCDB3236D14ffA6824F5BA
+ManifestType: installer
+ManifestVersion: 1.0.0
```
Comment (ItzLevvie) [inline manifests/s/SagMeinenNamen/TypingTest/1.0.0.0/SagMeinenNamen.TypingTest.installer.yaml:12]:
```suggestion
ManifestVersion: 1.2.0
```

File: manifests/s/SagMeinenNamen/TypingTest/1.0.0.0/SagMeinenNamen.TypingTest.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+PackageIdentifier: SagMeinenNamen.TypingTest
+PackageVersion: 1.0.0.0
+PackageLocale: en-US
+Publisher: SagMeinenNamen
+PackageName: Typing Test
+License: MIT License
+Copyright: Copyright ©  2022
+ShortDescription: Play fun typing races with this little program. How fast can you type?
+ManifestType: defaultLocale
+ManifestVersion: 1.0.0
```
Comment (ItzLevvie) [inline manifests/s/SagMeinenNamen/TypingTest/1.0.0.0/SagMeinenNamen.TypingTest.locale.en-US.yaml:13]:
```suggestion
ManifestVersion: 1.2.0
```

File: manifests/s/SagMeinenNamen/TypingTest/1.0.0.0/SagMeinenNamen.TypingTest.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: SagMeinenNamen.TypingTest
+PackageVersion: 1.0.0.0
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.0.0
```
Comment (ItzLevvie) [inline manifests/s/SagMeinenNamen/TypingTest/1.0.0.0/SagMeinenNamen.TypingTest.yaml:8]:
```suggestion
ManifestVersion: 1.2.0
```

File: manifests/s/SagMeinenNamen/TypingTest/1.0.0.0/SagMeinenNamen.TypingTest.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
```
Comment (ItzLevvie) [inline manifests/s/SagMeinenNamen/TypingTest/1.0.0.0/SagMeinenNamen.TypingTest.installer.yaml:2]:
```suggestion
# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
```

File: manifests/s/SagMeinenNamen/TypingTest/1.0.0.0/SagMeinenNamen.TypingTest.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
```
Comment (ItzLevvie) [inline manifests/s/SagMeinenNamen/TypingTest/1.0.0.0/SagMeinenNamen.TypingTest.locale.en-US.yaml:2]:
```suggestion
# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
```

File: manifests/s/SagMeinenNamen/TypingTest/1.0.0.0/SagMeinenNamen.TypingTest.yaml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
```
Comment (ItzLevvie) [inline manifests/s/SagMeinenNamen/TypingTest/1.0.0.0/SagMeinenNamen.TypingTest.yaml:2]:
```suggestion
# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by SagMeinenNamen — DISMISSED** [review]

File: manifests/s/SagMeinenNamen/TypingTest/1.0.0.0/SagMeinenNamen.TypingTest.installer.yaml [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+
+PackageIdentifier: SagMeinenNamen.TypingTest
+PackageVersion: 1.0.0.0
+Installers:
+- Architecture: x86
+  InstallerType: portable
+  InstallerUrl: https://github.com/SagMeinenNamen/TypingTest/raw/main/TypingTest.exe
+  InstallerSha256: E477B4DA4729F733E24C5D9794D48AA36A5B967ECABCDB3236D14ffA6824F5BA
+ManifestType: installer
+ManifestVersion: 1.0.0
```
Comment (ItzLevvie) [inline manifests/s/SagMeinenNamen/TypingTest/1.0.0.0/SagMeinenNamen.TypingTest.installer.yaml:12]:
```suggestion
ManifestVersion: 1.2.0
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — DISMISSED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**